README.md

```diff
@@ -494,6 +494,7 @@ APIs
 
 ### Natural Language Processing
 
+- [Cloudmersive NLP APIs](https://cloudmersive.com/nlp-api) - Wide array of unified NLP APIs including language translation, rephrasing, sentiment analysis, hate speech detection, and more.
```

Comment:
@laurajbouchard thanks for submitting! Is this API free to use?

Reply:
@n0shake yes! There are paid options, but the base tier is free and gives access to 800 calls/month.